tooling-tests/src/test/resources/test-src/proto-files/data-types/empty_message_types.proto

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+syntax = "proto3";
+
+import "google/protobuf/wrappers.proto";
+
+service ServiceWithEmpty {
```

Comment:
```suggestion
service ServiceWithPredefinedNames {
```

Reply:
Updated in [9955d08](https://github.com/ballerina-platform/module-ballerina-grpc/pull/707/commits/9955d08e7e62c83d460bc8bfaff47df251e22528), [9c22b7b](https://github.com/ballerina-platform/module-ballerina-grpc/pull/707/commits/9c22b7b87e186d6c5a60c7442d429369ec1dc77b)